app/views/admin/users/index.html.erb

```diff
@@ -29,51 +29,52 @@
         <%= link_to "Clear", admin_users_path, class: "btn btn-default border border-dark" %>
         <%= f.submit "Search", class: 'btn btn-primary border' %>
       </div>
-    <% end %>
-  </div>
-  <div class="row justify-content-center body-row">
-    <table class = "table table-striped table-hover mt-2">
-      <thead class="bg-dark">
+    </div>
+  <% end %>
+</div>
+<div class="row justify-content-center body-row">
+  <table class = "table table-striped table-hover mt-2">
+    <thead class="bg-dark">
+      <tr class="h-50 body-row">
+        <th><%= sort_link(@q, :first_name, [:first_name, 'last_name asc'], 'Name') %></th>
+        <th><%= sort_link(@q, :email) %></th>
+        <th><%= sort_link(@q, :last_sign_in_at) %></th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @users.each do |user| %>
         <tr class="h-50 body-row">
-          <th><%= sort_link(@q, :first_name, [:first_name, 'last_name asc'], 'Name') %></th>
-          <th><%= sort_link(@q, :email) %></th>
-          <th><%= sort_link(@q, :last_sign_in_at) %></th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @users.each do |user| %>
-          <tr class="h-50 body-row">
-            <td>
-              <% if user.admin? %>&#128273;<% end %>
-              <%= user.first_name.to_s + ' ' + user.last_name.to_s %>
-            </td>
-            <td><%= user.email %></td>
-            <td><%= user.last_sign_in_at %></td>
-            <td>
-              <%= link_to '&#x270D;'.html_safe, edit_admin_user_path(user), class: 'btn btn-info', title: 'edit' %>
-              <%= link_to '&#x2716;'.html_safe, admin_user_path(user), data: {
-          turbo_method: :delete, turbo_confirm: 'Are you sure?' }, class: 'btn btn-danger', title: 'delete' %>
-              <% if user.blocked %>
-                <%= button_to 'unblock', unblock_admin_user_path(id: user.id), method: :patch, class: 'btn btn-default border border-dark', title: 'unblock' %>
-              <% else %>
-                <%= button_to 'block', block_admin_user_path(id: user.id), method: :patch, class: 'btn btn-default border border-dark', title: 'block' %>
-              <% end %>
-            </td>
-          </tr>
-        <% end %>
-        <tr class="h-50 body-row bg-light">
-          <td class="col-sm-6 mt-2">
+          <td>
+            <% if user.admin? %>&#128273;<% end %>
+            <%= user.first_name.to_s + ' ' + user.last_name.to_s %>
```

Comment:
Why name.to_s??
Lets make a method for name

Reply:
to_s so that if it is nil it does not give error that + can't be performed on NilClass